frontends/verilog/verilog_parser.y

```diff
@@ -2551,6 +2554,7 @@ gen_stmt:
 		exitTypeScope();
 		delete $3;
 		delete $7;
+		SET_AST_NODE_LOC(ast_stack.back(), @1, @7);
```

Comment:
So the location ends at the label rather than `TOK_END`?  I am not sure.  I don't have a strong preference either way, though.

Reply:
The label to me feels as much part of the construct as the end token

Reply:
Is it?  The label is just the label; the structure ends with the `TOK_END`...  Again, I don't have a strong preference here, but my opinion is that it should end with `TOK_END`'s location.